src/vma/vma_extra.h

```diff
@@ -36,6 +36,9 @@
 
 #include <stddef.h>
 #include <stdint.h>
+
+#include "utils/bullseye.h"
```

Comment:
@igor-ivanov, this `#include` is problematic, need to find a different way.
We install `vma_extra.h` with libvma.rpm and allow application to use this header file and later running/loading with libvma.so itself.
But these application will not see the rest of the VMA tree and header files like the `utils/bullseye.h` in order to compile correctly. Need to keep here ONLY system include files.